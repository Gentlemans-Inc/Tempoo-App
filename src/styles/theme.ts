export const darkTheme = {
  colors: {
    background: '#121212',
    error: '#FF453A',
    gray: '#72788A',
    lightGray: '#d1d1d6',
    mediumGray: '#8E8E98',
    placeholder: '#D1D1D6',
    primary: '#65AFEC',
    text: '#fff',
    topBackground: '#212121',
  },
};

export const lightTheme = {
  colors: {
    background: '#fff',
    error: '#FF453A',
    lightGray: '#d1d1d6',
    mediumGray: '#8E8E98',
    gray: '#72788A',
    placeholder: '#D1D1D6',
    primary: '#65AFEC',
    text: '#343434',
    topBackground: '#f1f1f1',
  },
};

const theme = {
  colors: {
    background: '#fff',
    error: '#FF453A',
    lightGray: '#d1d1d6',
    mediumGray: '#8E8E98',
    gray: '#72788A',
    placeholder: '#D1D1D6',
    primary: '#65AFEC',
    text: '#343434',
    topBackground: '#f1f1f1',
  },
};

export default theme;
