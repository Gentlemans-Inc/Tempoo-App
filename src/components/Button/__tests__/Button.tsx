import React from 'react';
import renderer from 'react-test-renderer';
import {Button} from '../index';
import theme from '../../../styles/theme';
import 'jest-styled-components';

const foo = () => {};

test('Button component', () => {
  const tree = renderer
    .create(<Button label="foo" onPress={foo} theme={theme} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
