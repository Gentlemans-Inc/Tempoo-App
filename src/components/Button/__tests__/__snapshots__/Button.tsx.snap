// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component 1`] = `
<View
  accessible={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#65AFEC",
      "borderRadius": 1000,
      "opacity": 1,
      "paddingVertical": 21.5,
      "width": "92%",
    }
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#fff",
          "fontSize": 32,
          "fontWeight": "bold",
        },
      ]
    }
  >
    foo
  </Text>
</View>
`;
